package.json

```diff
@@ -98,6 +98,6 @@
     "local-npm:start": "verdaccio --config test/electron/local-npm-config.yml --listen 0.0.0.0:5539",
     "local-npm:publish-all": "lerna publish \"$VERSION_IDENTIFIER\" --yes --force-publish --exact --no-push --no-git-reset --no-git-tag-version --registry 'http://0.0.0.0:5539'",
     "local-npm:publish-all-win32": "lerna publish %VERSION_IDENTIFIER% --yes --force-publish --exact --no-push --no-git-reset --no-git-tag-version --registry 'http://0.0.0.0:5539'",
-    "test:electron": "xvfb-maybe --auto-servernum -- cucumber-js test/electron/features"
+    "test:electron": "xvfb-maybe --auto-servernum -- cucumber-js test/electron/features/performance.feature --retry 3 --retry-tag-filter @flaky"
```

Comment:
I don't think you meant to put `performance.feature` on the end?  No other features will be run.

Reply:
🤦